Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/PDController.java

```diff
@@ -739,7 +773,12 @@ public void storeAction(Action action) {
     public void undoAction() {
         Action action = actionHistory.undoPop();
         if (action != null) {
-            action.undo();
+            try {
+                action.undo();
+            } catch (Exception e) {
+                // LOGGER.error("Error when undoing filter action. Error message: " + e.getMessage());
```

Comment:
Can this be removed, or better, uncommented?  If uncommented, include `ex` in the logging call.